apps/dashboard/src/components/onboarding/ApplyForOpCreditsModal.tsx

```diff
@@ -114,13 +115,7 @@ export const ApplyForOpCreditsModal: React.FC = () => {
             <Card position="relative">
               <Box position="absolute">
                 <Badge
-                  borderRadius="full"
-                  size="label.sm"
-                  px={3}
-                  bgColor={creditsRecord.color}
-                  color="#fff"
-                  textTransform="capitalize"
-                  fontWeight="bold"
+                  className={`font-bold text-white capitalize bg-[${creditsRecord.color}] rounded-full px-3`}
```

Comment:
You can't do dynamic styles like this with tailwind - tailwind styles are generated at compile time - use `style` prop here for setting background